### PR TITLE
[Idle Detection article] Add info about ended OT (Merge on Nov. 11, 2020)

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add info about ended OT.

### CODE REVIEW

File: src/site/content/en/blog/idle-detection/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: |
   interaction with the keyboard, mouse, screen, activation of a screensaver, locking of the screen,
   or moving to a different screen. A developer-defined threshold triggers the notification.
 date: 2020-05-18
-updated: 2020-10-02
+updated: 2020-11-11
```
Comment (tomayac) [inline src/site/content/en/blog/idle-detection/index.md:11]:
```suggestion
updated: 2020-12-03
```

**Comment by tomayac** [comment]
Sorry, this fell through the cracks. It's information about an ended Origin Trial. @jpmedley or @kaycebasques, please approve and merge this when you have a chance.

**Comment by jpmedley** [comment]
I requested changes that have not been addressed.

**Comment by tomayac** [comment]
> I requested changes that have not been addressed.

@jpmedley I don't see any change requests.

**Review by jpmedley — CHANGES_REQUESTED** [review]
@tomayac apparently I neglected to submit them.

File: src/site/content/en/blog/idle-detection/index.md [hunk 2]
```diff
@@ -64,16 +64,15 @@ can limit these calculations to moments when the user interacts with their devic
...
-
-{% include 'content/origin-trial-register.njk' %}
+{% Aside %}
+  An origin trial for this feature ran from Chrome&nbsp;84 to Chrome&nbsp;86.
+  We are still coordinating with our partners to figure out when they will be ready to deploy code
+  to a wider audience using this experiment.
+  We will not be enabling the origin trial in Chrome&nbsp;87 and will update this article
+  when a new milestone has been chosen, likely Chrome&nbsp;88.
+  Interested developers can continue to test out the API locally,
+  including the permission changes landed in Chrome&nbsp;87.
```
Comment (tomayac) [inline src/site/content/en/blog/idle-detection/index.md:74]:
```suggestion
  including the permission changes landed in Chrome&nbsp;87,
  by flipping the [flag](#enabling-via-chrome:flags).
```

File: src/site/content/en/blog/idle-detection/index.md [hunk 3]
```diff
@@ -64,16 +64,15 @@ can limit these calculations to moments when the user interacts with their devic
...
-### Register for the origin trial {: #register-for-ot }
-
-{% include 'content/origin-trial-register.njk' %}
+{% Aside %}
+  An origin trial for this feature ran from Chrome&nbsp;84 to Chrome&nbsp;86.
+  We are still coordinating with our partners to figure out when they will be ready to deploy code
+  to a wider audience using this experiment.
+  We will not be enabling the origin trial in Chrome&nbsp;87 and will update this article
+  when a new milestone has been chosen, likely Chrome&nbsp;88.
+  Interested developers can continue to test out the API locally,
```
[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/idle-detection/index.md:73]: Yes. I've added this info.

File: src/site/content/en/blog/idle-detection/index.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ description: |
   interaction with the keyboard, mouse, screen, activation of a screensaver, locking of the screen,
   or moving to a different screen. A developer-defined threshold triggers the notification.
 date: 2020-05-18
-updated: 2020-10-02
+updated: 2020-12-03
```
Comment (tomayac) [inline src/site/content/en/blog/idle-detection/index.md:11]:
```suggestion
updated: 2020-12-07
```

**Comment by tomayac** [comment]
> @tomayac apparently I neglected to submit them.

Addressed. PTAL. Thank you!

**Review by jpmedley — APPROVED** [review]

**Pull request merged**